Update README.md to mention Discord Server (#5)
django/dsf-working-groups: README.md

  - For most groups, having the group self-manage its membership by approving new members and electing new Chair/Co-Chairs is fine.
  - For more sensitive or powerful committees (e.g. Conduct, Grants), having the Board approve membership changes may be more appropriate.
- **Budget** - how much money does the WG need (either to spend directly, or to pass on in the form of grants or similar)?

> - **Where will discussions and activities take place?** - the default is for us to make you a mailing list (`your-cool-wg@djangoproject.com`) or create a private channel (`#your-cool-wg`) in the Django Discord server, but if you want to do something else let us know.

Reviewer: ```suggestion
- **Where will discussions and activities take place?** - the default is for us to make you a mailing list (`your-cool-wg@djangoproject.com`). We can also create a private text channel for you on the [Django Discord server](https://discord.gg/xcRH6mN4fa). If you want to do something else let us know.
```
